src/ctap2/commands/get_assertion.rs

```diff
@@ -717,7 +695,9 @@ pub mod test {
             token_binding: Some(TokenBinding::Present(String::from("AAECAw"))),
         };
         let assertion = GetAssertion::new(
-            client_data.clone(),
+            CollectedClientDataWrapper::new(client_data)
```

Comment:
Do we still need `CollectedClientDataWrapper`?
I think we can also remove that and simply have `CollectedClientData` with a `hash()`-function.

Reply:
Yes, we can remove it.